app/controllers/repositories_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class RepositoriesController < ApplicationController
   before_action :authenticate_user!, except: [:show, :state, :state_with_date]
-  before_action :project_owner?, only: [:new, :create]
+  before_action :project_owner?, only: [:new, :create], if: "!params[:project_id].nil?"
```

Comment:
It's a matter of taste. If it bothers you, go for it!

Em qua, 24 de jun de 2015 às 11:34, marcheing <notifications@github.com>
escreveu:

> In app/controllers/repositories_controller.rb
> <https://github.com/mezuro/prezento/pull/235#discussion_r33153348>:
>
> > @@ -2,7 +2,7 @@
> >
> >  class RepositoriesController < ApplicationController
> >    before_action :authenticate_user!, except: [:show, :state, :state_with_date]
> > -  before_action :project_owner?, only: [:new, :create]
> > +  before_action :project_owner?, only: [:new, :create], if: "!params[:project_id].nil?"
>
> Can't you use proc { code } or a method here, instead of a string? It's
> always good to make sure code looks like what it is, instead of being
> inside strings :)
>
> —
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/mezuro/prezento/pull/235/files#r33153348>.
>